relation_engine_server/utils/arango_client.py

```diff
@@ -82,14 +82,16 @@ def create_collection(name, config):
         'keyOptions': {'allowUserKeys': True},
         'name': name,
         'type': collection_type,
-        'numberOfShards': num_shards
+        'numberOfShards': num_shards,
+        'waitForSync': True,
```

Comment:
The waiting for arangodb functionality was not working in other repos. This was one blind attempt to fix that, but didn't fix anything. It might still be useful to leave in though :shrug: